src/index.html

```diff
@@ -1,13 +1,66 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+<html lang="en" class="card-page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body class="block">
+    <div class="product-card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        class="product-card__image"
+      >
+
+      <div class="product-card__content">
+        <p class="product-card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p class="product-card__code">
+          Product code: 195434
+        </p>
+
+        <div class="product-card__rating">
+          <div class="stars stars--active">
```

Comment:
it should be not active, but count of active stars, like `stars--4`, and it should also work if I want to change count to `5`
```suggestion
          <div class="stars stars--4">
```